Week7/WEEK-7 SOLUTION.py

```diff
@@ -0,0 +1,62 @@
+import csv
+with open ("c://python//product.csv",'w') as f:
+    data=csv.writer(f)
+    header=["prod_no","prod_name","jan","feb","mar","apr","may","jun"]
+    data.writerow(header)
+    for i in range(12):
+        prod_no=int(input("Enter product no:"))
+        prod_name=input("Enter name:")
+        jan=int(input("Enter product selling of jan:"))
+        feb=int(input("Enter product selling of feb:"))
+        mar=int(input("Enter product selling of mar:"))
+        apr=int(input("Enter product selling of apr:"))
+        may=int(input("Enter product selling of may:"))
+        jun=int(input("Enter product selling of jun:"))
+        record=[prod_no,prod_name,jan,feb,mar,apr,may,jun]
+        data.writerow(record)
+
+with open("c://python//product.csv",'r')as f1:
+    content=csv.reader(f1)
+    for i in content:
+        print(i)
+#2.Create dataframe.        
+import pandas as pd
+df=pd.read_csv("c:\python3\product.csv")
+print(df)
+# 3. Change Column Name Product No, Product Name, January, February, March, April, May, June.
+df.columns=["Product_no","Product_name","January","February","March","April","May","June"]
+print(df)

```

Comment:
explain me the way you changes the column name?

Reply:
Using df.columns=["NewName1", "NewName2", ...], the code renames DataFrame columns by assigning a list of new names.